### ISSUE TITLE
Unable to build "master" branch

### ISSUE THREAD

**Issue by SchweinDeBurg** [doxygen/doxygen#7268]
```
[ 74/290] Building CXX object src/CMakeFiles/doxycfg.dir/__/generated_src/configimpl.cpp.o
FAILED: src/CMakeFiles/doxycfg.dir/__/generated_src/configimpl.cpp.o
/usr/bin/g++  -DYY_BUF_SIZE=262144 -DYY_READ_BUF_SIZE=262144 -I../qtools -I../libmd5 -I../liblodepng -I../libmscgen -I../libversion -I../vhdlparser -I../src -Igenerated_src -I/usr -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -m64 -march=native -mtune=native -mno-aes  -w -I/usr/include/llvm-8.0 -O2 -g -DNDEBUG -fvisibility=hidden -fvisibility-inlines-hidden   -D_GNU_SOURCE -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D__STDC_LIMIT_MACROS -std=gnu++14 -MD -MT src/CMakeFiles/doxycfg.dir/__/generated_src/configimpl.cpp.o -MF src/CMakeFiles/doxycfg.dir/__/generated_src/configimpl.cpp.o.d -o src/CMakeFiles/doxycfg.dir/__/generated_src/configimpl.cpp.o -c generated_src/configimpl.cpp
configimpl.l:1915:10: fatal error: configimpl.l.h: No such file or directory
```

**Comment by albert-github** [comment]
Working on a problem...
Workaround:
- make -i
- make

**Comment by albert-github** [comment]
Separately reported problem:
```
[  3%] Generating ../generated_src/mscgen_lexer.l.h
   File "/home/softadm/doxygen-git/doxygen/src/scan_states.py", line 22
     sys.exit('Usage: %s <lex_file>' % sys.argv[0])
                                                  ^
TabError: inconsistent use of tabs and spaces in indentation
make[2]: *** [libmscgen/CMakeFiles/mscgen.dir/build.make:74:
generated_src/mscgen_lexer.l.h] Error 1
make[2]: *** Deleting file 'generated_src/mscgen_lexer.l.h'
make[1]: *** [CMakeFiles/Makefile2:195:
libmscgen/CMakeFiles/mscgen.dir/all] Error 2
make: *** [Makefile:163: all] Error 2
```

**Comment by albert-github** [comment]
I've just pushed a proposed patch, pull request #7269

**Comment by albert-github** [comment]
Code has been integrated in master on github (please don't close the issue as this will be done at the moment of a release).

**Comment by SchweinDeBurg** [comment]
Confirmed, I've just successfully build RPM package from "master" branch under Fedora 26. Many thanx!

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.8.17.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
